[python3] Build interpreter (#14891)
microsoft/vcpkg: docs/maintainers/vcpkg_build_make.md

The target passed to the make build command (`./make <target>`). If not specified, the 'all' target will
be passed.

### INSTALL_TARGET

> The target passed to the make build command (`./make <target>`) if `ENABLE_INSTALL` is used. If not specified,
> the 'install' target will be passed.

Reviewer: ```suggestion
The target passed to the make build command (`./make <target>`) if `ENABLE_INSTALL` is used. Defaults to 'install'.
```

As written this is confusing because "if not specified" looks like it goes with `ENABLE_INSTALL`. How about this?

(And while we're here maybe the above should say "defaults to 'all'" to keep parallel structure but I won't ask you to edit code that isn't green :) )

Author: This looks like a reasonable suggestion. We'll just have to wait for CI to rebuild everything again 😆
